### PR TITLE
Revert "fix: user avatar status indicator position (#3967)"

### PR DESCRIPTION
**Pull request by yqrashawn**
This reverts commit f73f3e9f8260ce7fcd1a6c3c363a64ee13fb2713.

revert #3967 it requires mobile inputs and the mobile PR is not merged yet

### CODE REVIEW

**Comment by yqrashawn** [comment]
@pavloburykh  revert PR here, can you approve this? (sorry to @ wrong person)

**Review by pavloburykh — APPROVED** [review]

**Review by qfrank — APPROVED** [review]

**Comment by pavloburykh** [comment]
> @pavloburykh revert PR here, can you approve this? (sorry to @ wrong person)

@yqrashawn approved. But in case it needs review - please request additional review from some of devs.

Also, could you please rebase and update go branch in corresponding mobile PR https://github.com/status-im/status-mobile/pull/17110 so we can test it? We will take it into work once it will be reviewed and approved.

**Pull request merged**
